scripts/spec.py

```diff
@@ -0,0 +1,52 @@
+from datetime import date
+import json
+
+import bokeh.models as bm; bm
+import bokeh.models.widgets as bmw; bmw
+
+from bokeh.model import Model
+
+def _proto(obj, defaults=False):
+    return json.dumps(obj.to_json(defaults), sort_keys=True, indent=None)
+
+data = {}
+for name, m in sorted(Model.__class__.model_class_reverse_map.items()):
+    item = {
+        'name'  : name,
+        'bases' : [str(base)[8:-2] for base in m.__bases__],
```

Comment:
`base.__module__ + '.' + base.__name__` or is there a reason to use code that I needed to run to know what it does?